src/monitor/pg_auto_failover.c

```diff
@@ -75,6 +76,11 @@ StartMonitorNode(void)
 {
 	BackgroundWorker worker;
 
+	DefineCustomBoolVariable("pgautofailover.enable_version_checks",
+							 "Enable extension version compatiblity checks",
+							 NULL, &EnableVersionChecks, true, PGC_SIGHUP,
```

Comment:
searching through this PR I came to the conclusion this variable is not read ever. What is the use of this GUC? should we have a check for the value of `EnableVersionChecks ` in `checkPgAutoFailoverVersion` to blindly continue when this GUC is set to false?

Reply:
Good spot! now fixed.